src/app/+detail/index.ts

```diff
@@ -17,7 +17,6 @@ export const routes = [
     Detail
   ],
   imports: [
-    BrowserModule,
     FormsModule,
```

Comment:
I have included it again. But `BrowserModule` is already imported in `app.module.ts`.